dan-stineback/gulpfile.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const gulp = require('gulp');
+const webpack = require('webpack-stream');
+const clean = require('gulp-clean');
+
+const paths = {
+  js: __dirname + '/app/**/*.js',
+  html: __dirname + '/app/**/*.html'
+  // css: __dirname + '/app/**/*.css'
+};
+
+gulp.task('clean', ()=>{
+  return gulp.src('./build/*', {read:false})
+    .pipe(clean());
+});
+
+gulp.task('copy-html', ['clean'], ()=>{
+  return gulp.src(paths.html)
+    .pipe(gulp.dest('./build'));
+});
+
+// gulp.task('copy-css', ['clean'], ()=>{
```

Comment:
Turn this on or remove the link to css from index.html.  I'm seeing a 404 on the css file when I load the page.